apps/client/src/pages/IntroPage.js

```diff
@@ -5,14 +5,26 @@ import { useHistory } from "react-router-dom";
 import Layout from "../components/Layouts/DefaultLayout";
 import Loading from "../components/Loading";
 import Nav from "../components/Nav";
+import { actions, eventNames, sections } from "../config/matomo";
 import withTopic from "../hoc/withTopic";
+import withTracking from "../hoc/withTracking";
 import { geturl, routes } from "../routes";
 
-const IntroPage = ({ topic }) => {
+const IntroPage = ({ matomoTrackEvent, topic }) => {
   const history = useHistory();
   const { text, intro } = topic;
   const Intro = React.lazy(() => import(`../intros/${intro}`));
 
+  const goToNext = () => {
+    matomoTrackEvent({
+      action: actions.CLICK_INTERNAL_NAVIGATION,
+      category: topic.name.toLowerCase(),
+      name: `${eventNames.FORWARD} ${sections.LOCATION_INPUT}`,
+    });
```

Comment:
Why do we need a `trackEvent` here? It's just a regular `pageView` handled by the Router right?

Reply:
true! removed it